app/views/admin/loan_questions/_form.slim

```diff
@@ -16,9 +16,21 @@
 
     = f.input :data_type, as: :select, label: t('loan_questions.data_type'),
       collection: @data_types
-      
+
     = f.input :has_embeddable_media, label: t('loan_questions.has_embeddable_media')
 
+    label Required for which loan types?
+
+    - if @loan_question.parent
+      = f.collection_radio_buttons :override_associations,
+        [[false, 'Inherits from parent'], [true, 'Overrides parent']], :first, :last,
+        item_wrapper_class: 'radio'
+
+    - hideLoanTypes = @loan_question.parent && !@loan_question.override_associations ? 'hidden': ''
+    .loan-types-container class="#{hideLoanTypes}"
+      = f.input_field :loan_type_ids, collection: @loan_type_options, multiple: true,
+        class: "loan-types"
```

Comment:
Looks like this is not getting I18n'd either?

**Important**

SimpleForm gives you a super easy way to I18n a select like this. There are already some examples in `en.yml`:

```
simple_form:
    labels:
      project_step_move:
        move_type: "Which Date?"
        shift_subsequent: "Shift Others?"
    options:
      person:
        owning_division_role:
          admin: "Admin"
          member: "Member"
          none: "None"
      project_step_move:
        move_type:
          change_date: "Change step's scheduled date"
          mark_completed: "Mark step completed on this date"
```

See also https://github.com/plataformatec/simple_form#i18n

We should always be using this.